src/components/BetaBanner.js

```diff
@@ -16,14 +18,18 @@ const BetaBanner = () => (
       </svg>
       <span className="bold">This site is in beta.</span>{' '}
       Help us make it better:{' '}
-      <a
-        className="bold white border-bottom"
-        href="https://github.com/18F/crime-data-explorer/issues"
+      <button
+        className='bg-transparent bold border-none border-bottom cursor-pointer underline white'
```

Comment:
did you try adding the `btn` class here? (i think that brings along several of these things, like the transparent background, cursor, etc.)